docs/about/history.md

```diff
@@ -0,0 +1,3 @@
+### 2.0.0
+
+* Initial release 
```

Comment:
I think we need at least _some_ history of previous releases. A preferred alternative would be ditching history altogether and track everything via GitHub releases instead. [1.1.0](https://github.com/spekframework/spek/releases/tag/v1.1.0) is a good example. Otherwise there will be a dichotomy between local history and GitHub releases — and most likely a need of copy-pasting changes there and there.

Reply:
Yep, it's not actually being used - forgot to remove it.